envisage/ids.py

```diff
@@ -0,0 +1,62 @@
+# (C) Copyright 2007-2019 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only
+# under the conditions described in the aforementioned license.  The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+# Thanks for using Enthought open source!
+
+from envisage.core_plugin import CorePlugin
+from envisage.plugins.ipython_kernel.ipython_kernel_plugin import (
+    IPythonKernelPlugin
+)
+from envisage.plugins.python_shell.python_shell_plugin import PythonShellPlugin
+from envisage.ui.single_project.project_plugin import ProjectPlugin
+from envisage.ui.tasks.tasks_plugin import TasksPlugin
+from envisage.ui.workbench.workbench_plugin import WorkbenchPlugin
+
+#### Extension Points #########################################################
+
+# Extension Points defined by the CorePlugin.
+CLASS_LOAD_HOOKS = CorePlugin.CLASS_LOAD_HOOKS
```

Comment:
Let's drop this; we don't want to encourage its use, and we don't want another place where we might have to deprecate / remove.